src/snowcli/cli/streamlit/commands.py

```diff
@@ -40,36 +40,6 @@
 add_init_command(app, project_type="Streamlit", template="default_streamlit")
 
 
-@app.command("list")
-@with_output
-@global_options_with_connection
-def streamlit_list(**options) -> CommandResult:
```

Comment:
I think we forgot to delete show/drop/desc methods from StreamlitManager. Shouldn't we?

Reply:
Good point, removed them